src/test/java/core/basesyntax/FruitServiceTest.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitBox;
+import core.basesyntax.service.*;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitServiceTest {
+    private FruitFileReader reader = new FruitFileReader();
+    private FruitService service = new FruitService();
+
+    @Test(expected = RuntimeException.class)
+    public void testWrongOperation() {
+        service.parseOperations(reader.readFruitFile("src/WrongOperation.csv"));
```

Comment:
let's move all of these files to this package "src/test/resources/"

Reply:
done